### DISCUSSION TITLE
Point system on mee6 / Tea portal

### DISCUSSION THREAD

**Discussion by rav2525** [teaxyz/white-paper#140] [Q&A]
Will the community points on mee6 currently to transferred to a better place, for instance on tea portal itself?
Also currently the points are reflected very late on mee6 (for ex. I haven't see last weeks activity points until today), can there be automated process instead that can speed up this process.

CommuniTea is the best!

**Comment by mrn1ck13** [comment]
I was also wondering the same thing as it seems it isn't updated super frequently. Hopefully we can find out soon!

**Reply by rav2525** [reply]
yeah, I hope. When it takes too much time, you loose patience because it makes you think - all the time you spent is finally worth it or not.

**Reply by iman136938** [reply]
yes good idea

**Reply by rav2525** [reply]
thanks
